var/spack/repos/builtin/packages/py-bglibpy/package.py

```diff
@@ -13,6 +13,7 @@ class PyBglibpy(PythonPackage):
     git = "ssh://bbpcode.epfl.ch/sim/BGLibPy"
 
     version('develop', branch='master')
+    version('4.1.2', commit='d01c9359edf92e39be5b840b5bb975da1c960988')
```

Comment:
Tags?

Reply:
Please, let's not start this discussion agaaiin ... We need this fairly urgently...